fcrepo-auth-common/src/main/java/org/fcrepo/auth/common/FedoraUserSecurityContext.java

```diff
@@ -15,7 +15,7 @@
  */
 package org.fcrepo.auth.common;
 
-import static org.fcrepo.kernel.api.FedoraJcrTypes.JCR_CONTENT;
+import static org.modeshape.jcr.api.JcrConstants.JCR_CONTENT;
```

Comment:
I agree with @acoburn on this, and I ask him to please create a ticket for that refactoring to get dependence on `org.fcrepo.kernel.modeshape.rdf.impl ` out of the kernel. That was a gross oversight for which I think I can mostly be blamed. @cbeer wrote that, but I convinced him that it was a good idea. It was a good idea, but not good enough. We need to completely rethink the RDF generation subsystem in light of these things:

1. The new Java 8 Streams API
2. Our desire to refactor JCR into invisibility.
3. Our need to produce a specification that can be reimplemented in a practical way.

Let me further suggest. along the lines of @acoburn 's argument, that this module actually be renamed ` fcrepo-auth-common-modeshape`.